.github/workflows/test.yaml

```diff
@@ -95,7 +95,7 @@ jobs:
             pip install .[test]
 
         - name: Run Tests
-          run: pytest -m "not fuzzing" -s --cov=src -n auto --dist loadgroup
+          run: pytest -m "not (fuzzing or manual)" -s --cov=src -n auto --dist loadgroup
```

Comment:
@mikeshultz the new custom marker is used here and it is also registered in pyproject.toml. it is not a direct pytest feature but just a pattern ive used in the past for similar scenarios.

Reply:
I've been wondering what the `-m` option was.  Interesting, thanks.